src/components/Modal/Modal.js

```diff
@@ -46,21 +46,22 @@ export default function Modal(props){
     let sizeVariant = setSize(size);
 
     let visibility;
```

Comment:
You should write this as:
```ts
const visibility = show ? 'show' : 'hide';
```
Even then, I have no idea why you are using `boolean` and `string` to describe the same thing